data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/schema/MySqlConnectionManager.java

```diff
@@ -10,7 +10,7 @@
 import java.sql.SQLException;
 import java.util.Properties;
 
-public class ConnectionManager {
+public class MySqlConnectionManager {
```

Comment:
Consider creating ConnectionManager interface and implementation for MySql/Postgres ConnectionManager

Reply:
Created the interface.